ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -29,6 +30,6 @@
     version: "{{ version }}"
 
 - name: install the catalog from the catalog location
-  shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ catalog_namespace }} {{ cli.path }} chdir="{{ catalog_location }}/packages"
+  shell: ./installCatalog.sh {{ catalog_auth_key }} {{ api_host }} {{ catalog_namespace }} {{ cli.path }} {{ whisk_api_host }} chdir="{{ catalog_location }}/packages"
```

Comment:
@csantanapr what is the harm in creating a new variable to encapsulate exactly what you have here inline? If there is some harm caused, then we should get rid of all the variables, shouldn't we? (I'm not actually suggesting this, but just trying to get at the root of your reasoning)

I would suppose that the benefit of creating the variable is that it makes this (already rather long line) easier to read and understand. To justify putting the value inline would require that any harm caused by creating the variable should be greater than the benefit of increased readability.